### PR TITLE
Added a New Algorithm Wahaj Raza 

### PR DESCRIPTION
**Pull request by Wahaj-Raza**
Thanks for reading my PR

### CODE REVIEW

**Review by Sabine91 — CHANGES_REQUESTED** [review]
Well done with documentation and everything just a few more things, 
- If you are inserting code bits in a markdown file make sure they are formatted properly
- Try to add comments in your code for each block, and add proper spacing so as to avoid congestion.

Solution:
- Re do the Markdown file to appear as intended.
- Add proper formatting in your source code like a line of space between the functions, and adding comments for each block.

File: Algorithms/Dynamic Programming/LCM/LCM.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# LCM
```
Comment (shhossain) [inline Algorithms/Dynamic Programming/LCM/LCM.md:1]:
```suggestion
# Least Common Multiple(LCM)
```

Change this.

File: Algorithms/Dynamic Programming/LCM/LCM.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+## Introduction
+The least common multiple (L.C.M.) of two numbers is the smallest positive integer that is perfectly divisible by the two given numbers.
+
+For example, the L.C.M. of 12 and 14 is 84.
+
+Program to Compute LCM
+# Python Program to find the L.C.M. of two input number
```
Comment (shhossain) [inline Algorithms/Dynamic Programming/LCM/LCM.md:8]:
```suggestion
### Python Program to find the L.C.M. of two input number
```

And this.

**Review by shhossain — CHANGES_REQUESTED** [review]
And add time complexity of LCM for your code.

**Review by Sabine91 — APPROVED** [review]
Well done

**Comment by Sabine91** [comment]
Just add the time complexity and will close the request.

**Pull request merged**